nextcord/embeds.py

```diff
@@ -434,13 +444,46 @@ def set_image(self: E, url: MaybeEmpty[Any]) -> E:
                 del self._image
             except AttributeError:
                 pass
+            finally:
+                self._local_image = None
         else:
             self._image = {
                 "url": str(url),
             }
 
         return self
 
+    def set_local_image(
+        self: E,
+        image_path: Union[str, bytes, os.PathLike, io.BufferedIOBase],
+        *,
+        image_name: Optional[str] = None,
```

Comment:
Instead of both of these, why not make a single parameter of type `File` similar to other places.

Also related: #607 (can make it take `Union[bytes, Asset, Attachment, File]`)